main.js

```diff
@@ -71,6 +80,9 @@ class Field {
       if (gameOverReason !== null) {
         console.log(gameOverReason);
         break;
+      } else if (this.gameIsWon === true) {
```

Comment:
This PR seems more complicated than it needs to be. It could be just this:

```suggestion
      } else if (this.field[this.playerYPosition][this.playerXPosition] === "^") {
```

Now you don't need `gameIsWon` or `checkIfGameIsWon()`.